internal/parser/scanner/rule_based_scanner_test.go

```diff
@@ -46,7 +47,7 @@ func TestRuleBasedScanner(t *testing.T) {
 			},
 		},
 		{
-			"SELECT      FROM || & +7 59 \"foobar\"",
+			"SELECT      FROM || & +7 5.9 \"foobar\"",
```

Comment:
Don't change test cases, add new ones

Reply:
Was testing, will revert and add new when final.